source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -26,80 +32,96 @@ responsive: true
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility, Compiler Secrets and Modern Ember @ EmberCamp 🐹](#section-url)
+
+
+Last week the Ember community went on another extraordinary trip into the metropolitan wilderness of Illinois:
+[**EmberCamp 2019**](https://embercamp.com) took place in Chicago on September 16th and was insightful, exciting and entertaining as always!
+
+With twelve, versed speakers, the schedule offered thoughtful insights into:
+
+- what's new in Ember with its [first edition, Octane](https://emberjs.com/editions/octane/)
+- how [Embroider](https://github.com/embroider-build/embroider) and Ember will literally build a future together
+- everything you need to know to build **accessible JavaScript apps** in 2019
```

Comment:
"everything" sounds cool, but I think an accurate summary may be that the community learned how to write more semantic HTML to improve their app's accessibility. What are your thoughts?

Reply:
Yeah, I think this is an accurate summary for Marie's talk, but I was thinking that points in this list here might also expand on bits and pieces of several talks, instead of describing  a single specific one. E.g. the audience learnt about accessibility concerns for giving control over animation playback to the user in James' talk, or  Matt's talk was highlighting how different map library solutions are more or less accessible and developers should take this into consideration when choosing one

What are your thoughts?

Reply:
Ah, I see. I didn't consider the possibility that we actually learned about accessibility from more than one talk. Yep, what you wrote makes sense now, thanks!